### PR TITLE
surahs & chapters

### PR DESCRIPTION
**Pull request by AlaaAttya**
Added: chapters listing
Added: surahs listing

We need to know where will be the chapters be placed and how will the filters will go on

### CODE REVIEW

File: src/actions/chapters.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  type: FETCH_CHAPTERS,
+  data
+});
+
+export const getChapters = () => {
+  const url = 'https://quranicaudio.com/api/sections';
+
+  return dispatch => fetch(url)
+    .then(response => response.json())
+    .then(data => {console.log("yes");dispatch(chapters(data))})
```
Comment (thabti) [inline src/actions/chapters.js]: remove console.log here, might be worth us adding a logger like `redux-logger`

**Review by thabti — APPROVED** [review]

File: src/container/Chapters/index.js [hunk 2]
```diff
@@ -1,16 +1,135 @@
...
+// import { Component } from 'react';
+// import { Text, View } from 'react-native';
 
-export default class Chaters extends Component {
+// export default class Chaters extends Component {
+//   static navigationOptions = {
+//     title: ({ state }) => `${state.params.name}`
+//   };
+//   render() {
+//     const data = this.props.navigation.state.params.data;
```
Comment (hosainnet) [inline src/container/Chapters/index.js]: Do we need this commented out code?

**Review by hosainnet — COMMENTED** [review]
Remove commented out code, but otherwise LGTM

**Comment by AlaaAttya** [comment]
done

**Comment by thabti** [comment]
Let's remove surahs and stick to chapters

**Comment by AlaaAttya** [comment]
@sabeurthabti so what will be the action when user chooses a chapter?

**Comment by thabti** [comment]
@AlaaAttya selectChapter ? ping me on slack.

**Pull request merged**